interface-definitions/service_monitoring_telegraf.xml.in

```diff
@@ -0,0 +1,76 @@
+<?xml version="1.0"?>
+<interfaceDefinition>
+  <node name="service">
+    <children>
+      <node name="monitoring">
+        <properties>
+          <help>Monitoring services</help>
+          <priority>1280</priority>
+        </properties>
+        <children>
+          <node name="telegraf" owner="${vyos_conf_scripts_dir}/service_monitoring_telegraf.py">
+            <properties>
+              <help>Service telegraf</help>
+            </properties>
+            <children>
+              <node name="authentication">
+                <properties>
+                  <help>Authentication parameters</help>
+                </properties>
+                <children>
+                  <leafNode name="login">
+                    <properties>
+                      <help>Login</help>
```

Comment:
Please add a:

```
<valueHelp>
  <format>txt</format>
  <description>Login is used for foo bar baz</description>
</valueHelp>
```

Is there a restriction in login names? Please check for a regex pattern.

Reply:
Login was replaced with "organization"